crosscat/utils/inference_utils.py

```diff
@@ -90,12 +85,12 @@ def mutual_information(M_c, X_Ls, X_Ds, Q, n_samples=1000):
         Linfoot.append(Linfoot_sample)
 
 
-    assert(len(MI) == len(Q))
-    assert(len(Linfoot) == len(Q))
+    assert len(MI) == len(Q)
+    assert len(Linfoot) == len(Q)
 
     return MI,  Linfoot
 
-def calculate_MI_bounded_discrete(X, Y, M_c, X_L, X_D):
+def calculate_MI_bounded_discrete(X, Y, M_c, X_L, _X_D):
```

Comment:
I disagree.  Intentionally unused variables are very common in higher order languages, because lots of interfaces insist on giving you things even if you don't want them.  "unused_" is much too long a prefix to standardize on for something that happens so often; I have personally been very happy with single leading underscore for that purpose.  I suggest codifying that convention in pylint, as Venturecxx does.

In case you are worried about conflicting with the Python convention for private members, I think it doesn't, because intentionally unused functions and fields basically never happen, and if they for some reason did, they might as well be private.